install_python_module.sh

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash -x
+
+# Installs shapeworks Python module
+
+if [ "$#" -lt 1 ]; then
+    echo "Installs python module binary (shapeworks_py) into conda, optionally adding necessary runpaths to the library."
+    echo "Usage: $0 <shapeworks_py_library> [<lib_dir_1> ... <lib_dir_n>]"
+    exit 1
+fi
+
+# install pip module
+if ! pip install Python/shapeworks; then exit -1; fi
+
+PYTHON_LIBRARY=$1; shift
+CONDA_INSTALL_DIR=`pip show shapeworks | grep Location | awk '{print $2}'`/shapeworks
+
+# copy shapeworks_py library to conda install dir
+cp "${PYTHON_LIBRARY}" "${CONDA_INSTALL_DIR}"
+PYTHON_LIBRARY="${CONDA_INSTALL_DIR}/`basename ${PYTHON_LIBRARY}`"
+
+# update runpaths for shapeworks_py, compiled portion of package
+if [[ "$(uname)" == "Linux" ]]; then
+
+    # if unspecified, use package path from github action (see package.sh)
+    if [ "$#" -lt 1 ]; then
+        if [[ "$PR_NUMBER" != "" ]]; then
+            VERSION="ShapeWorks-PR-${PR_NUMBER}-linux"
+        else
+            VERSION="ShapeWorks-$(git describe --tags)-linux"
+        fi
+	      LIBDIR="`pwd`/package/${VERSION}/lib"
+
+        # Special case for when we are on the master branch (dev releases)
+        BRANCH=$(git rev-parse --abbrev-ref HEAD)
+        if [[ "$BRANCH" == "master" ]]; then
```

Comment:
What's the special handling for master doing here?  I don't see that the NUMBER, HASH, or VERSION are used after this.  Can this section be removed?